### DISCUSSION TITLE
Galaxy brain Badge

### DISCUSSION THREAD

**Discussion by ghost** [community/community#112973] [New to GitHub] [answered]
Hi there ,
GitHub has anounced that they removed badge rewarding mechanism from community discussions (not sure exactly it is or something else) I wanted to know is there any other way that make people able to obtain Galaxy brain badge ??

**Comment by ParasRupani** [comment]
Hey, I don't believe that the Galaxy Brain has been removed from the badges list.
You can check the entire [List of Badges here.](https://github.com/drknzz/GitHub-Achievements?tab=readme-ov-file)
Take note of the following:
- Answer at least 2 discussions.
- You CANNOT mark your own answers to your own questions. They have to be marked by another user.
- Discussions should be in a public repository.
There's a current bug observed by some users whereby marking your own answers can prevent you from getting the badge instead.

Hope this helps.

**Reply by CodingwithFawad** [reply]
Thank you

**Comment by Akash1134** [comment, marked as answer]
We made the decision to disable the ability to earn Achievements in our Community in order to discourage users from participating in coordinated or inauthentic activity like rapid questions and answers in order to earn badges. You can learn more about this decision in our announcement post here [Achievements will no longer be available in the Community](https://github.com/orgs/community/discussions/106536).

Note that GitHub's [Acceptable Use Policies](https://docs.github.com/en/site-policy/acceptable-use-policies/github-acceptable-use-policies) prohibits coordinated or inauthentic activity like rapid questions and answers.

**Reply by ghost** [reply]
so isn't there any other way that i can get galaxy brain badge ?

**Reply by CodingwithFawad** [reply]
Thank you Brother

**Comment by Abdulaziz-developer1** [comment]
I also think that it is to answer a discussion
(get an accepted answer)

PLS help me to get that accept my answer.
![image](https://github.com/user-attachments/assets/6b08766b-dc41-4a0d-b01d-333dce7b032c)
